app/src/androidTest/java/org/kiwix/kiwixmobile/main/TopLevelDestinationTest.kt

```diff
@@ -31,14 +31,14 @@ class TopLevelDestinationTest {
     topLevel {
       clickReaderOnBottomNav {
       }
-      clickLibraryOnBottomNav {
-        clickFileTransferIcon {
-        }
-      }
       clickDownloadOnBottomNav {
         clickOnGlobeIcon {
         }
       }
+      clickLibraryOnBottomNav {
+        clickFileTransferIcon {
+        }
+      }
```

Comment:
I was suspecting this to cause some errors, as on real physical device with internet connection on, the download screen keeps on making call to the network which in turn slowers the testing process and hence causes clicks and back presses occur in no time. Therefore swapped it with Library, the safest area as on Reader it kept on showing toast and snackbar each time. The issue may not be occur at CI but is visible in real devices.